x-pack/plugins/security_solution/public/common/containers/local_storage/use_messages_storage.tsx

```diff
@@ -6,7 +6,7 @@
  */
 
 import { useCallback } from 'react';
-import { useKibana } from '../../lib/kibana';
+import { useKibana } from '../../../common/lib/kibana';
```

Comment:
no need?

Reply:
It may be, but want to talk to @XavierM about it. It's related to your comment here: https://github.com/elastic/kibana/pull/94840#discussion_r596401518

We currently pass this storage via the `StartServices` in security solution's plugin.tsx, I believe, so that all views have access to the same storage. `storage` actually isn't a service that can be listed in `kibana.json` just a util we made into a service via the changes in the plugin.tsx. @XavierM would just pulling it here be okay?